lib/ui/classical_components/cropping_custom_ui.dart

```diff
@@ -0,0 +1,157 @@
+import 'package:flutter/material.dart';
+import 'package:scanbot_sdk/classical_components/classical_cropping.dart';
+import 'package:scanbot_sdk/json/common_data.dart' as common;
+import 'package:scanbot_sdk/scanbot_sdk.dart';
+
+/// This is an example screen of how to integrate new classical barcode scanner component
+class CroppingScreenWidget extends StatefulWidget {
+  CroppingScreenWidget({Key? key, required this.page}) : super(key: key);
+  common.Page page;
+
+  @override
+  _CroppingScreenWidgetState createState() => _CroppingScreenWidgetState();
+}
+
+class _CroppingScreenWidgetState extends State<CroppingScreenWidget> {
+  /// this stream only used if you need to show live scanned results on top of the camera
+  /// otherwise we stop scanning and return first result out of the screen
+  bool showProgressBar = false;
+  bool licenseIsActive = true;
```

Comment:
licenseIsActive is always true there is no verifying if it is actually active or not